app/models/container_quota_item.rb

```diff
@@ -1,3 +1,25 @@
 class ContainerQuotaItem < ApplicationRecord
   belongs_to :container_quota
+
+  virtual_column :quota_desired_display,  :type => :string
+  virtual_column :quota_enforced_display, :type => :string
+  virtual_column :quota_observed_display, :type => :string
+
+  def quota_desired_display
+    quota_display(quota_desired)
+  end
+
+  def quota_enforced_display
+    quota_display(quota_enforced)
+  end
+
+  def quota_observed_display
+    quota_display(quota_observed)
+  end
+
+  private
+
+  def quota_display(quota)
+    (quota % 1).zero? ? quota.to_i.to_s : quota.to_s
+  end
```

Comment:
I think these changes were part of another PR?  These changes are [already in master](https://github.com/ManageIQ/manageiq/blob/56745dbdcc92dccb12c6299134a19c6fbc0321c6/app/models/container_quota_item.rb).

I can't figure out why this is not giving a merge conflict...